### PR TITLE
Hometask day8 lists

### PR DESCRIPTION
**Pull request by DenisDav95**
I have finished sub-task 3-7 and done some fixes

### CODE REVIEW

File: src/main/java/homework/day8/Cars.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        BufferedWriter logs = new BufferedWriter(new FileWriter("cars.txt"));
+
+        for (String i : cars) {
+            logs.write("\"" + i + "\"");
+            logs.newLine();
+        }
+        logs.close();
+
+        for (int i = 0; i < cars.size(); i++) {
+            if (cars.get(i).length() > 4) {
```
Comment (nina-zhukava) [inline src/main/java/homework/day8/Cars.java]: skips one жигуль, you need to rewrite this if (add i-- if condition is true)

**Review by nina-zhukava — CHANGES_REQUESTED** [review]

File: src/main/java/homework/day8/Cars.java (hunk 1 above)
Comment (DenisDav95) [reply src/main/java/homework/day8/Cars.java]: Fixed

File: src/main/java/homework/day8/Cars.java [hunk 2]
```diff
@@ -29,8 +29,9 @@ public static void main(String[] args) throws IOException {
         logs.close();
 
         for (int i = 0; i < cars.size(); i++) {
-            if (cars.get(i).length() > 4) {
+            if ((cars.get(i).length() > 4)) {
```
Comment (nina-zhukava) [inline src/main/java/homework/day8/Cars.java:32]: why did you add more parentheses?)

**Review by nina-zhukava — APPROVED** [review]
approved, but please remove added parentheses

File: src/main/java/homework/day8/Cars.java (hunk 2 above)
Comment (DenisDav95) [reply src/main/java/homework/day8/Cars.java:32]: It was overcoding to check that if return'true'
I will delete it
